application/apps/protocol/binding/wasm/src/msg/attachment.rs

```diff
@@ -0,0 +1,11 @@
+use crate::*;
+use extend::extend;
+use prost::Message;
+use serde_wasm_bindgen::{from_value, to_value};
+use wasm_bindgen::prelude::*;
```

Comment:
I think we can at some point include the use statements inside the macro itself by writing the full paths to the types inside it.
This would make any future changes easier by changing the macro itself and not having to add using statement for each file at a time.